src/main.rs

```diff
@@ -403,6 +411,27 @@ mod tests {
                 .trim()
                 .to_string()
         }
+
+        fn check_console_device_id(&self) -> bool {
+            self.ssh_command("cat /sys/bus/pci/devices/*/device | grep 0x1043")
+                .trim()
+                .to_string()
+                .is_empty()
+        }
+
+        fn check_console_vendor_id(&self) -> bool {
+            self.ssh_command("cat /sys/bus/pci/devices/*/vendor | grep 0x1af4")
```

Comment:
Not sure about these. I think you really need to look for the vendor+device pair together.

Reply:
Valid concern. It has been taken care of. Thanks.